site/en/workflows.md

```diff
@@ -1217,19 +1244,146 @@ copies them the a new Output fileGrp, re-generating the PAGE XML from the curren
+Schwabacher) or font style (e.g. *italic*, **bold**).
+
+**Note:** `ocrd-tesserocr-fontshape` can either use existing segmentation or segment on-demand. It can detect the following font styles:
+  * `fontSize`
+  * `fontFamily`
+  * `bold`
+  * `italic`
+  * `underlined`
+  * `monospace`
+  * `serif`
+
+**Note:** `ocrd-typegroups-classifier` can only annotate font families on page level but can detect a wider variety of fonts, including the confidence value (separated by colon). Supported `fontFamily` values:
+  * `Antiqua`
+  * `Bastarda`
+  * `Fraktur`
+  * `Gotico`-Antiqua
+  * `Greek`
+  * `Hebrew`
+  * `Italic`
+  * `Rotunda`
+  * `Schwabacher`
+  * `Textura`
+  * `other_font`
+  * `not_a_font`
+
+**Note:** `ocrd-typegroups-classifier` only works on *non-binarized* RGB images.
+
+#### Available processors
+
+<table class="processor-table">
+  <thead>
+    <tr>
+      <th>Processor</th>
+      <th>Parameter</th>
+      <th>Remarks</th>
+    <th>Call</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr data-processor="ocrd-tesserocr-fontshape">
+      <td>ocrd-tesserocr-fontshape</td>
+      <td><code>-P model osd -P padding 2</code></td>
+      <td></td>
+      <td><code>ocrd-tesserocr-fontshape -I SEG-WORD -O SEG-WORD-FONT</code></td>
+    </tr>
+    <tr data-processor="ocrd-typegroups-classifier">
+      <td>ocrd-typegroups-classifier</td>
+      <td><code>-P network /path/to/densenet121.tgc</code></td>
+      <td>Download [`densenet121.tgc` from GitHub](https://github.com/seuretm/ocrd_typegroups_classifier/raw/master/ocrd_typegroups_classifier/models/densenet121.tgc)</td>
+      <td><code>ocrd-typegroups-classifier -I OCR-D-IMG -O OCR-D-IMG-FONTS</code></td>
+    </tr>
+  </tbody>
+
+<!-- END-EVAL -->
+
 # Recommendations
 
 <!-- BEGIN-INCLUDE ./repo/ocrd-website.wiki/Workflow-Guide-recommendations.md -->
-All processors, with the exception of those for post-correction, were tested on
-selected pages of some prints from the 17th and 18th century.
+In order to facilitate the usage of OCR-D and the configuration of workflows, we provide two workflows
+which can be used as a start for your OCR-D-tests. They were determined by testing the processors listed
+above on selected pages of some prints from the 17th and 18th century.
 
 The results vary quite a lot from page to page. In most cases, segmentation is a problem.
 
-These recommendations may also work well for other prints of those centuries.
-
 Note that for our test pages, not all steps described above werde needed to obtain the best results.
 Depending on your particular images, you might want to include those processors again for better results.
 
+We are currently working on regression tests with the help of which we will be able to provide more profound
+workflows soon, which will replace those interm solutions. 
+
+## Minimal workflow
+
+Since `ocrd-tesserocr-recognize` can do all line segmentation, region
+segmentation, table detection and even binarization (Otsu), just like the
```

Comment:
```suggestion
Since `ocrd-tesserocr-recognize` can do all binarization (Otsu), region
segmentation, table recognition, line segmentation and text recognition at once, just like the
```